Scripts/Winwing/fslabs_winwing_cdu.py

```diff
@@ -100,7 +100,7 @@ async def run_mobiflight_websocket_client():
                 fontName = "AirbusThales"
                 await mobi_websocket_connection.send(f'{{ "Target": "Font", "Data": "{fontName}" }}')
                 logging.info(f"Setting font: {fontName}")
-
+                await asyncio.sleep(1) # wait a second for font to be set
             await asyncio.sleep(0.05)
```

Comment:
Duplicate sleep. This could be increased to 1 second and the preceding sleep removed